src/components/WelcomeBox.css

```diff
@@ -16,7 +16,17 @@
 }
 
 html .welcomebox .toggle-button-end {
-  bottom: 11px;
   position: absolute;
   top: auto;
+  right: 0;
```

Comment:
hmm - i think we can take advantage of `postcss-bidirection` here

https://github.com/gasolin/postcss-bidirection#absolute-positioning-example